jetty-http/src/main/java/org/eclipse/jetty/http/HttpURI.java

```diff
@@ -449,12 +480,12 @@ public URI toURI()
          */
         private static final Index<Boolean> __ambiguousSegments = new Index.Builder<Boolean>()
             .caseSensitive(false)
-            .with("%2e", Boolean.TRUE)
-            .with("%2e%2e", Boolean.TRUE)
-            .with(".%2e", Boolean.TRUE)
-            .with("%2e.", Boolean.TRUE)
-            .with("..", Boolean.FALSE)
-            .with(".", Boolean.FALSE)
+            .with("%2e", Boolean.TRUE)    // Is real dot segment not removed by normalisation.
```

Comment:
I'd stick with American spelling and use `normali*z*ation` instead.

Also, the comments are not very clear. 

This is an `Index.Builder`, so it should just reference that -- no external references to normalization or such.

To me the comment should be: "`// %2e is considered ambiguous.` -- while I cannot understand what does "Is real dot segment not removed by normalisation." mean.

If I lookup `%2e` in this `Index` I get back `true` -- that's it.

Reply:
I was asked by Greg to put these comments in. Should I just remove these comments? I don't think "`// %2e is considered ambiguous.`" adds much over just reading the code.

Reply:
I have removed the comments here.